packages/esm-patient-banner-app/src/banner-tags/print-identifier-sticker-content.component.tsx

```diff
@@ -0,0 +1,66 @@
+import React, { forwardRef, useEffect, useImperativeHandle, useRef } from 'react';
+import { useConfig } from '@openmrs/esm-framework';
+import { type ConfigObject } from '../config-schema';
+import IdentifierSticker from './print-identifier-sticker.component';
+import styles from './print-identifier-sticker-content.scss';
+
+interface PrintIdentifierStickerContentProps {
+  labels: Array<{}>;
+  numberOfLabelColumns: number;
+  numberOfLabelRowsPerPage: number;
+  patient: fhir.Patient;
+}
+
+const PrintIdentifierStickerContent = forwardRef<HTMLDivElement, PrintIdentifierStickerContentProps>(
+  ({ labels, numberOfLabelColumns, numberOfLabelRowsPerPage, patient }) => {
+    const { printIdentifierStickerWidth, printIdentifierStickerHeight, printIdentifierStickerPaperSize } =
+      useConfig<ConfigObject>();
+
+    const maxLabelsPerPage = numberOfLabelRowsPerPage * numberOfLabelColumns;
+    const pages: Array<typeof labels> = [];
+
+    for (let i = 0; i < labels.length; i += maxLabelsPerPage) {
+      pages.push(labels.slice(i, i + maxLabelsPerPage));
+    }
+
+    if (numberOfLabelColumns < 1 || numberOfLabelRowsPerPage < 1 || labels.length < 1) {
+      return;
+    }
+
+    return (
+      <div>
+        <style type="text/css" media="print">
+          {`
+            @page {
+              size: ${printIdentifierStickerPaperSize};
+            }
+            @media print { html, body { height: initial !important; overflow: initial !important; background-color: white; }}
+          `}
+        </style>
```

Comment:
@denniskigen, this will be fixed in https://openmrs.atlassian.net/browse/O3-3870

Reply:
We have a `copyStyles` prop from the `react-to-print` lib, won't that suffice this?
Thanks!